src/DynamoCore/Core/NodeModelAssemblyLoader.cs

```diff
@@ -199,6 +199,45 @@ internal static bool ContainsNodeModelSubType(Assembly assem)
             return assem.GetTypes().Any(IsNodeSubType);
         }
 
+        internal static bool ContainsNodeModelSubTypeReflectionLoaded(Assembly assem, Type nodeModelType = null)
+        {
+            if(nodeModelType is null)
+                return assem.GetTypes().Any(x => IsNodeSubType(x));
+            
+            return assem.GetTypes().Any(x => IsNodeSubTypeReflectionLoaded(x, nodeModelType));
+        }
+
+        internal static bool IsNodeSubTypeReflectionLoaded(Type t, Type nodeModelType)
```

Comment:
Yea so the issue with that is i need to make sure that its using a reference to the same type. because im calling this method with a type coming form the MLC context, i need to also compare it with a type from the MLC. Doing `typeof(NodeModel)` will get me the NodeModel type from the normal context (not sure what the right definition is of that 🤡) and thoese two dont point to the same refernce, so it will always return false.

Another option is to compare the name, but dont think that's as safe as this, what do you think?

Reply:
ah, gotcha, makes sense, didn't know the same type would not equal itself if it's from different contexts (MLC vs normal)